dbi/dbi.go

```diff
@@ -76,14 +77,11 @@ func (dbiPlg *DbiPlugin) CollectMetrics(mts []plugin.PluginMetricType) ([]plugin
 		return nil, err
 	}
 
-	hostname, _ := os.Hostname()
-
 	for _, m := range mts {
-		if value, ok := data[joinNamespace(m.Namespace())]; ok {
-			metric := plugin.PluginMetricType{
+		if value, ok := data[m.Namespace().String()]; ok {
+			metric := plugin.MetricType{
 				Namespace_: m.Namespace(),
 				Data_:      value,
-				Source_:    hostname,
 				Timestamp_: time.Now(),
 				Version_:   m.Version(),
 			}
```

Comment:
I have a question what about tags - is there should added also: `Tags_: m.Tags()` or not. What is the best approach?